### PR TITLE
ADD: emotion API

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

**Comment by madisonmay** [comment]
Pretty simple addition, would appreciate a lightning code review before merge though :)

File: indicoio/text/emotion.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+       >>> text = "I did it. I got into Grad School. Not just any program, but a GREAT program. :-)"
+       >>> possible = indicoio.emotion(text)
+       >>> emotion = possible.keys()[np.argmax(possible.values())]
+       >>> probability = np.max(possible.values())
+       >>> 'Predicted `%s` with probability %.4f' % (emotion, probability)
+       u'Predicted `joy` with probability 0.7744'
+
+    :param text: The text to be analyzed.
+    :type text: str or unicode
+    :rtype: Dictionary of language probability pairs
```
Comment (sihrc) [inline indicoio/text/emotion.py]: language > emotion

File: tests/test_remote.py [hunk 2]
```diff
@@ -82,6 +83,13 @@ def test_batch_political(self):
         response = political(test_data, api_key=self.api_key)
         self.assertTrue(isinstance(response, list))
 
+    def test_batch_emotion(self):
+        test_data = ["I did it. I got into Grad School. Not just any program, but a GREAT program. :-)"]
+        response = emotion(test_data, api_key=self.api_key)
```
Comment (sihrc) [inline tests/test_remote.py]: pass in apikey?

Comment (sihrc) [inline tests/test_remote.py]: we don't wanna do this right?

Comment (madisonmay) [inline tests/test_remote.py]: all of our client libs support this style of input

Comment (madisonmay) [inline tests/test_remote.py]: and that's the general format used by tests in this file

Comment (madisonmay) [inline tests/test_remote.py]: it definitely should be a valid option

Comment (sihrc) [inline tests/test_remote.py]: is it popped

Comment (sihrc) [inline tests/test_remote.py]: the other non batch doesn't pass in the key

Comment (madisonmay) [inline tests/test_remote.py]: I'll remove it then

**Comment by aidankmcl** [comment]
Birdperson is pleased

**Pull request merged**